kahuna/public/js/image/view.html

```diff
@@ -1,7 +1,7 @@
 <gr-top-bar>
     <gr-top-bar-nav>
-        <a class="top-bar-item side-padded clickable" ui:sref="search" title="Back to search">
-            <gr-icon-label gr-icon="search">Search</gr-icon-label>
+        <a class="top-bar-item side-padded clickable" ui:sref="search" gr:tooltip="Back to search" gr:tooltip-position="right">
```

Comment:
Not really - removed. The `title` attribute will still display in the corner case mentioned.